### ISSUE TITLE
MLPDropoutLayer seems has no hidden layer ?

### ISSUE THREAD

**Issue by memeda** [yoonkim/CNN_sentence#2]
Hi ,  recently I am reading the code anf founed it seems that : At MLPDroupoutLayer , there is no hidden layer added . 
First , hidden_unites  defined here : 

conv_net_sentence.py - 311line : hidden_units=[100,2] 

and changed here : 

conv_net_sentence.py - 93 line : hidden_units[0] = feature_maps*len(filter_hs)

so it steel just has 2 elements . 

And at conv_net_classes.py : 
96 line :  self.weight_matrix_sizes = zip(layer_sizes, layer_sizes[1:])
103 : next_dropout_layer_input = _dropout_from_layer(rng, input, p=dropout_rates[0])
105 : for n_in, n_out in self.weight_matrix_sizes[:-1]:

and here self.weight_matrix_sizes[:-1] is a empty list . So there seems no hidden layer defined ?

**Comment by yoonkim** [comment]
Right, we go directly from the CNN output to the softmax, without any hidden layers.
